mt940/models.py

```diff
@@ -381,7 +381,7 @@ def parse(self, data):
                     self.transactions.append(transaction)
                 else:
                     transaction.data.update(result)
-            elif tag.scope is Transaction and self.transactions:
```

Comment:
Ok... disregard my comments completely. I was looking at the diff in reverse. Great patch :)